common/src/lib/java/dev/engine_room/flywheel/lib/material/Materials.java

```diff
@@ -10,43 +10,37 @@
 public final class Materials {
 	public static final Material SOLID_BLOCK = SimpleMaterial.builder()
 			.build();
-	public static final Material SOLID_UNSHADED_BLOCK = SimpleMaterial.builder()
+	public static final Material SOLID_UNSHADED_BLOCK = SimpleMaterial.builderOf(SOLID_BLOCK)
```

Comment:
Is adding new static final field in the middle binary compat safe?

Reply:
How a `static final` field is computed should never impact binary compat. Right now, no new fields are added, but adding new fields should not affect binary compat either as they are referenced by name.